oshi-core/src/main/java/oshi/software/common/AbstractOperatingSystem.java

```diff
@@ -116,11 +116,11 @@ public List<OSProcess> getChildProcesses(int parentPid, Predicate<OSProcess> fil
         // Get this pid and its children
         List<OSProcess> childProcs = queryChildProcesses(parentPid);
         // Extract the parent from the list
-        OSProcess parent = childProcs.stream().filter(p -> p.getProcessID() == parentPid).findAny().orElse(null);
+        OSProcess parent = childProcs.stream().parallel().filter(p -> p.getProcessID() == parentPid).findAny().orElse(null);
```

Comment:
In theory could improve performance, finding any in parallel... vs. finding any in a single thread... but may end up doing extra work too.  Would have to benchmark this.  It's just an object getter though, and a relatively small number (child processes are usually in the single digits) so I'd say no here.